### DISCUSSION TITLE
NASA發射DART探測器撞擊小行星

### DISCUSSION THREAD

**Discussion by delia07152** [1101-GS4719-C/Python-Community-GS4719-C#780] [General]
https://technews.tw/2022/10/13/hera-mission-dimorphos-asteroid-orbit/

在太空中，有數以千計的天體，雖然地球有大氣層的保護，但能無法保證小行星能在此完全被消滅，光是直徑10公尺的小行星就能造成小範圍的建築物受損，所以天文學家必須不斷地觀察，並找出哪顆可能會撞上地球。NASA在去年發射DART探測器，使其撞擊小行星以改變公轉軌道，並於今年九月底成功撞上，而目前已確定已經離開可能的威脅路徑，這是人類史上第一次利用主動撞擊小行星防禦。希望這項技術能越來越純熟，有朝一日人類能完全不受威脅。

**Comment by danwan1027** [comment]
人類的技術越來越進步，未來又可以減少一項天然災害的危害了！

**Comment by Pietra1226** [comment]
不知道衝擊小行星軌道會不會有什麼其他的影響

**Comment by AnAnistired** [comment]
好像也有一篇一樣的了，但還是感謝分享

**Comment by evanchou1204** [comment]
感覺真實世界正在向科幻電影慢慢靠攏，好神奇

**Comment by Tuoresa** [comment]
好驚人，原來這件事人類已經可以成功做到了。科技的發展逐漸實現好多太空電影裡的想像，好期待未來更多發展

**Comment by ru0372572** [comment]
恐龍滅絕也是因為小行星撞擊，也許發展這項科技有望阻止下一次人類大滅絕(?
